services/motion/motion.go

```diff
@@ -181,11 +181,23 @@ type Service interface {
 	) ([]PlanWithStatus, error)
 }
 
+// ObstacleDetectorConfig is the protobuf version of ObstacleDetector
+type ObstacleDetectorConfig struct {
+	VisionService string `json:"vision_service"`
+	Camera        string `json:"camera"`
+}
+
+// ObstacleDetector pairs a vision service with a camera, informing the service about which camera it may use
+type ObstacleDetector struct {
+	VisionService resource.Name
```

Comment:
This is probably related to our conversation in the other comment thread, but why are these not `vision.Service` and `camera.Camera`?